test/tests/functional/pbs_nonprint_characters.py

```diff
@@ -947,18 +944,22 @@ def test_terminal_control_hook_name(self):
         """
         # Create hook
         hook_name = "h%s%sd" % (self.bold, self.red)
-        a = {'event': "execjob_begin", 'enabled': 'True', 'debug': 'True'}
-        try:
-            rv = self.server.create_hook(hook_name, a)
-        except PbsManagerError:
+        cmd = ['qmgr', '-c', 'create hook %s' % hook_name]
```

Comment:
create cmd1 for creating hook and cmd2 for deleting and use it at all places instead of over-writing cmd all the time.

Reply:
Done.